cmake/Options.cmake

```diff
@@ -14,6 +14,9 @@ cmake_dependent_option(TRIBOL_ENABLE_DOCS "Enables Tribol Docs" ON "ENABLE_DOCS"
 option(TRIBOL_USE_SINGLE_PRECISION "Use single-precision floating point" OFF)
 option(TRIBOL_USE_64BIT_INDEXTYPE "Use 64-bit index type" OFF)
 
-option(TRIBOL_ENABLE_SIDRE "Enables use of Sidre" OFF)
-option(TRIBOL_ENABLE_SLIC "Enables use of Slic" ON)
```

Comment:
Is this not necessary anymore?

Reply:
I verified they are not used anywhere except where I deleted.